tools/monitoring/mysql/README.md

```diff
@@ -0,0 +1,33 @@
+# Monitoring Database
+
+The monitoring database is a MySQL instance on Cloud SQL. It stores the error logs for each prow job
+failures; and the alert status for all error patterns.
+
+## Database Connection With Cloud Shell
+
+Navigate to SQL in Google Cloud Platform. Click on the knative-monitoring instance. In the Overview tab,
```

Comment:
is this the "monitoring" cluster mentioned in "Create the cluster"? Or are you talking about the existing one in knative-tests?

Reply:
The instance here is referring to the Cloud SQL Instance which is named 'knative-monitoring'. This is different from the cluster instance.